MicroserviceDemo/applications/AuthServer.Host/AuthServerDataSeeder.cs

```diff
@@ -146,15 +148,15 @@ private async Task CreateClientsAsync()
                 "console-client-demo",
                 new[] { "BloggingService", "IdentityService", "InternalGateway", "ProductService", "TenantManagementService" },
                 new[] { "client_credentials", "password" },
-                commonSecret,
+                commonSecret.Sha256(),
```

Comment:
The `commonSecret` is already secret.

`const string commonSecret = "E5Xd4yMqjP5kjWFKrYgySBju6JVfCzMyFp7n2QmMrME=";`

Reply:
If commonsecret is encrypted, you'd better add a comment, otherwise it will be misunderstood